packages/plugins/typed-variable-modal/src/TypedVariableModal.js

```diff
@@ -73,7 +73,7 @@ export class TypedVariableModal extends Modal {
     /**
      * An array holding arrays with the name of the type and the display name
      *     for the type. Ex: [['Penguin', 'PENGUIN'], ['Giraffe', 'GIRAFFE']].
-     * @type {Array<Array<string>>}
+     * @type {Array.<Array.<string>>}
```

Comment:
Why was this changed? Based on [this](https://github.com/google/closure-compiler/wiki/Types-in-the-Closure-Type-System) it seems like the "." is deprecated syntax.